shared/utils/helpers.js

```diff
@@ -57,3 +58,14 @@ export const prettifyNameSingular = (resourceName, resourceType) => {
   const resources = prettifyNamePlural(resourceName, resourceType);
   return resources.slice(0, -1);
 };
+
+export const getErrorMessage = (error, message = null) => {
+  let errorNotification = message
+    ? message
+    : 'An error occured. The component cannot be rendered.';
+
+  if (error?.message && typeof error?.originalMessage !== 'object') {
```

Comment:
what's the 'originalMessage'?

Reply:
When we get a message it is stringified. So we need to check if it was an object or string before and that's what the originalMessage is for.